Dockerfile

```diff
@@ -17,10 +17,11 @@ ARG KUBECTL_VERSION="v1.18.2"
 
 COPY --from=build /k9s/execs/k9s /bin/k9s
 RUN apk add --update ca-certificates \
-  && apk add --update -t deps curl vim \
```

Comment:
I think nano should be added here prior the cleaning up the dependencies. Or am I missing it?

Reply:
I've moved out the editor installing after `apk del --purge deps`  to make sure they stay in the image after temporary deps  packages are purged.

Actually, I'm not sure how vim stay in the image after it was installed with `apk add --update -t deps` and after  `apk del --purge deps`  is invoked. It supposed to be purged just like curl.